.github/workflows/functional-test.yaml

```
@@ -0,0 +1,60 @@
+name: Functional test for sandbox image
+
+on:
+  pull_request:
+    paths:
+      - docker/**
+      - charts/**
+
+defaults:
+  run:
+    # https://github.com/actions/runner/issues/241#issuecomment-577360161
+    shell: 'script -q -e -c "bash {0}"'
+
+jobs:
+  run-tests:
+    name: Functional test
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.8
+      - uses: whoan/docker-build-with-cache-action@v5.11.0
+        with:
+          image_name: flyteorg/flyte-sandbox
+          image_tag: dind
```

just to make it obvious, can you rename this to:
```suggestion
          image_tag: local
```